aep/general/0156/aep.yaml

```
@@ -0,0 +1,7 @@
+---
+id: 156
+state: reviewing
+slug: resource-design
```

```suggestion
slug: singletons
```

Slugs are intended to replace numbers as the primary way of referring to AIPs (still a WIP).  They also turn into a short link; this one will be aep.dev/singletons.